### PR TITLE
Motor reorder and direction dialogs: hardcode throttle value to 6%

### PR DESCRIPTION
**Pull request by limonspb**
Problem:
if a pilot is using dynamic idle, he can set dshot idle as low as he wants, for example, 1%.
In this case, motor direction dialog will spin motors at 1010, and this won't spin motors at all.

Second problem:
motor reorder dialog spins motors at 1200 always, which is a bit too high for average drones.

Dirty solution (until we fully implement this solution: https://github.com/betaflight/betaflight-configurator/pull/2766)
Spin motors at 6% in both motor direction and motor reorder dialogs regardless of the settings.
And hope it will cover most of the cases.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
I checked that reasonably DShot values are sent to the motors.
One small detail confused me - when changing direction, the motors spin continuously, but when re-ordering, they spin intermittently (on for 250ms, off for 500ms).  It confused me at first.
Maybe it would be better for the motors to spin continuously during both tests?

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
I'll approve so as to not delay an RC3 Configurator release over the small issue of the intermittent motor pattern, since the motor pattern can be changed, if we want to, in a different PR.

**Review by Asizon — APPROVED** [review]

**Comment by Asizon** [comment]
@limonspb @haslinghuis why we cant go with this https://github.com/betaflight/betaflight-configurator/pull/2766 ??

**Comment by limonspb** [comment]
> @limonspb @haslinghuis why we cant go with this #2766 ??

yes i think we can...

**Pull request merged**
